api/envoy/config/route/v3/route.proto

```diff
@@ -106,6 +106,17 @@ message RouteConfiguration {
   // option. Users may wish to override the default behavior in certain cases (for example when
   // using CDS with a static route table).
   google.protobuf.BoolValue validate_clusters = 7;
+
+  // The maximum bytes of the response :ref:`direct response body
+  // <envoy_api_field_config.route.v3.DirectResponseAction.body>` size. If not specified the default
+  // is 4096.
```

Comment:
You probably want to update `docs/root/intro/arch_overview/http/http_routing.rst` that references 4kb and put in a RST link to this field.

Reply:
Updated.